community/i3-gaps/build

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -e
+
+# export V=1 for verbose output when building
+[ "${V:-0}" -eq 1 ] > /dev/null 2>&1 && set -x || :
+
+export CHOST="${CHOST:-x86_64-pc-linux-musl}"
+
+PATCHES="${PATCHES:-}"
+
+PATCHES="
+i3-4.16-musl-GLOB_TILDE.patch
+${PATCHES}
+"
+
+if [ -n "${PATCHES}" ]
+then
+	for patch in ${PATCHES}
+	do
+		if [ -z "${patch}" ] || [ ! -e "${patch}" ]
+		then
+			continue
+		fi
+		patch -p1 < "${patch}"
+	done
+fi
```

Comment:
Everything above this line can go. Patches should be done in a for loop if all `-p` match or one per line.